CMakeLists.txt

```diff
@@ -11,6 +11,10 @@ set(CMAKE_C_STANDARD_REQUIRED ON)
 set(LPAC_CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 list(APPEND CMAKE_MODULE_PATH ${LPAC_CMAKE_MODULE_PATH})
 
+if(CMAKE_EXPORT_COMPILE_COMMANDS STREQUAL "")
+  set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
indent problem, other code is 4 spaces, but this is 2 spaces

Reply:
Fixed.